### DISCUSSION TITLE
How do I reload labwc?

### DISCUSSION THREAD

**Discussion by tuzbtla** [labwc/labwc#2387] [Q&A]
When I change /etc/xdg/labwc/rc.xml, then I execute labwc -r, which returns 00:00:00.000 [../src/main.c:72] LABWC_PID not set.

Now the touch screen, when I swipe directly, selects text. But I need to scroll through the window, is there a configuration that allows me to simulate a mouse wheel, like a two-finger swipe?

**Comment by 01micko** [comment]
`killall -s SIGHUP labwc` will do it but I wonder why LABWC_PID isn't set.

**Reply by tuzbtla** [reply]
Thanks for your reply. I don't know. I just burned the latest image of Raspberry PI 5.

**Comment by tuzbtla** [comment]
I have solved the problem. I set the environment variables myself.
